dependency-check-suppressions.xml

```diff
@@ -64,6 +66,7 @@
     <vulnerabilityName>CVE-2019-14540</vulnerabilityName>
     <vulnerabilityName>CVE-2019-16335</vulnerabilityName>
     <vulnerabilityName>CVE-2019-17267</vulnerabilityName>
+    <vulnerabilityName>CVE-2019-10172</vulnerabilityName>
```

Comment:
I wish we could figure out why dependency-check thinks we have jackson-mapper-asl on the classpath. I have run `mvn dependency:tree` several times before and I don't see it anywhere.

Reply:
Is Avro or something shading the jackson 1.x classes and dependency-check is smart enough to find those?

Reply:
Looks like

```
[INFO] +- org.apache.avro:avro:jar:1.7.7:compile
[INFO] |  +- org.codehaus.jackson:jackson-core-asl:jar:1.9.13:compile
[INFO] |  +- org.codehaus.jackson:jackson-mapper-asl:jar:1.9.13:compile
[INFO] |  +- com.thoughtworks.paranamer:paranamer:jar:2.3:compile
[INFO] |  +- org.xerial.snappy:snappy-java:jar:1.0.5:compile
[INFO] |  \- org.apache.commons:commons-compress:jar:1.4.1:compile
[INFO] |     \- org.tukaani:xz:jar:1.8:compile
```